Network/Socket.hs

```diff
@@ -401,6 +398,10 @@ import Network.Socket.Buffer hiding (
     recvBufMsg,
     sendBufMsg,
     sendBufTo,
+
+    -- * Initialisation on Windows
+    withSocketsDo,
```

Comment:
This is a wrong position. It should be in front of `where`. I will fix this by myself.